Userland/Applications/Help/main.cpp

```diff
@@ -221,23 +229,23 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     };
 
     page_view->on_link_click = [&](auto& url, auto&, unsigned) {
-        if (url.protocol() != "file") {
-            open_external(url);
-            return;
-        }
-        auto path = Core::File::real_path_for(url.path());
-        if (!path.starts_with("/usr/share/man/")) {
+        if (url.protocol() == "file") {
```

Comment:
What is actually being changed here? Opening programs with a file URL from Help works on master, I just checked.

Reply:
If it's in preparation for the second commit, I don't really see the point of making the change in the first commit.

Reply:
The main issue with this is that `real_path_for` will cause an unveil violation for `/bin` paths, which I avoid hereby. Relative path support is removed as we don't need it with the new help://man URLs.

Reply:
But I agree, this should be its own commit.

Reply:
> The main issue with this is that real_path_for will cause an unveil violation for /bin paths, which I avoid hereby.

Got it (& should've checked my debug console :sweat_smile:). Legit change then, but please mention the reason in the commit!